### DISCUSSION TITLE
Center or not?

### DISCUSSION THREAD

**Discussion by jaka1-newbie** [ImageMagick/ImageMagick#3349] [Help]
Im 7.10, win64

convert D:\output.png -gravity center -rotate 90 -size 2500x85   -background #0005 -fill #C7601A -font Limit-corner caption:"Tales of Space and Time  H. G. Wells" -compose over -composite -rotate -90 D:\output.png

grey area - centered
font - not  

Where problem? 
![output](https://user-images.githubusercontent.com/67605008/110000519-103e3280-7d1c-11eb-8108-1edae8cd2ed8.png)

**Comment by fmw42** [comment]
Sorry, I do not understand your question.  Can you show an example of what you want?

**Comment by jaka1-newbie** [comment]
Tales of Space and Time H. G. Wells on grey area - not centered

**Reply by jaka1-newbie** [reply]
It is 1mm reight from center

**Reply by fmw42** [reply]
Please provide your input image called output.png and your font.  What is -background #5? You likely are using some special character. Does it work correctly with a different font?

**Reply by jaka1-newbie** [reply]
![output](https://user-images.githubusercontent.com/67605008/110004437-19c99980-7d20-11eb-97f7-704cab26faa0.png)
https://dropmefiles.com/tuP9V

I tried without this symbol - same problem

**Reply by jaka1-newbie** [reply]
-background # 0 0 0 5

**Reply by jaka1-newbie** [reply]
-background # 0 0 0 5 without spaces - github delete 000 from command

**Reply by jaka1-newbie** [reply]
Sorry! 
It is FONT problem
